docker-compose.yml

```diff
@@ -36,3 +46,10 @@ services:
       interval: 30s
       timeout: 10s
       retries: 3
+  mailer:
+    image: sj26/mailcatcher:latest
+    restart: always
+    expose:
+      - 1025
```

Comment:
this port is already exposed in the dockerfile of mailcatcher, what is the need to expose it here explicitly? Ok for me, I am just curious.

Reply:
> what is the need to expose it here explicitly?

Technically speaking, none.

As you said, it's already exposed in the mailcatcher docker image, so the only purpose of this is to make it clear what are the available ports for this container.

Reply:
ok fine with me :)